Gemfile

```diff
@@ -70,6 +72,7 @@ gem "sanger_sequencing", "~> 0.0.1", path: "vendor/engines/sanger_sequencing"
 gem "secure_rooms", path: "vendor/engines/secure_rooms"
 gem "split_accounts", "~> 0.0.1", path: "vendor/engines/split_accounts"
 gem "synaccess_connect", "0.2.2", github: "tablexi/synaccess"
+gem "net-telnet"
```

Comment:
Let's not worry about that for now. I may look at the synaccess gem at some point, because it would be nice to be able to get rid of that dependency.